libd-llvm/src/d/llvm/expression.d

```diff
@@ -468,6 +468,7 @@ struct ExpressionGen {
 			auto thisPtr = LLVMBuildExtractValue(builder, dg, m.hasContext, "");
 			auto vtblPtr = LLVMBuildStructGEP(builder, thisPtr, 0, "");
 			auto vtbl = LLVMBuildLoad(builder, vtblPtr, "vtbl");
```

Comment:
movl	$8, %edi
	callq	__d_allocmemory
	movq	%rax, %rbx
	movq	_C1t1A__vtblZ@GOTPCREL(%rip), %rax
	movq	%rax, (%rbx)
	movq	%rbx, %rdi
	callq	__D1t1A3fooFC1t1AZv
	movq	(%rbx), %rax
	movq	%rbx, %rdi
	popq	%rbx
	jmpq	*8(%rax)

vs

	movl	$8, %edi
	callq	__d_allocmemory
	movq	%rax, %rbx
	movq	_C1t1A__vtblZ@GOTPCREL(%rip), %rax
	movq	%rax, (%rbx)
	movq	%rbx, %rdi
	callq	__D1t1A3fooFC1t1AZv
	movq	(%rbx), %rax
	movq	%rbx, %rdi
	popq	%rbx
	jmpq	*8(%rax)

The difference is indeed striking.